CSS/Clase3-Tarea.css

```
@@ -0,0 +1,39 @@
+.Caja-Madre {
+    color: white;
+    background-color: #1b1b1b;
+    border-radius: 5px;
+    width: 1000px;
+    height: 760px;
+    font-family: BlinkMacSystemFont,"Segoe UI","Roboto","Oxygen","Ubuntu","Cantarell","Fira Sans","Droid Sans","Helvetica Neue",sans-serif;
+
+}
+
+.Caja-1 {
```

ejemplo de add suggestion
```suggestion
.caja-1 {
```